debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -298,37 +297,40 @@ DAY_MICROSECOND:                     'DAY_MICROSECOND';
 
 // JSON keywords
 JSON_ARRAY:                          'JSON_ARRAY';
-JSON_OBJECT:                         'JSON_OBJECT';
-JSON_QUOTE:                          'JSON_QUOTE';
+JSON_ARRAYAGG:                       'JSON_ARRAYAGG';
+JSON_ARRAY_APPEND:                   'JSON_ARRAY_APPEND';
+JSON_ARRAY_INSERT:                   'JSON_ARRAY_INSERT';
 JSON_CONTAINS:                       'JSON_CONTAINS';
 JSON_CONTAINS_PATH:                  'JSON_CONTAINS_PATH';
+JSON_DEPTH:                          'JSON_DEPTH';
 JSON_EXTRACT:                        'JSON_EXTRACT';
-JSON_KEYS:                           'JSON_KEYS';
-JSON_OVERLAPS:                       'JSON_OVERLAPS';
-JSON_SEARCH:                         'JSON_SEARCH';
-JSON_VALUE:                          'JSON_VALUE';
-JSON_ARRAY_APPEND:                   'JSON_ARRAY_APPEND';
-JSON_ARRAY_INSERT:                   'JSON_ARRAY_INSERT';
 JSON_INSERT:                         'JSON_INSERT';
+JSON_KEYS:                           'JSON_KEYS';
+JSON_LENGTH:                         'JSON_LENGTH';
 JSON_MERGE:                          'JSON_MERGE';
 JSON_MERGE_PATCH:                    'JSON_MERGE_PATCH';
 JSON_MERGE_PRESERVE:                 'JSON_MERGE_PRESERVE';
+JSON_OBJECT:                         'JSON_OBJECT';
+JSON_OBJECTAGG:                      'JSON_OBJECTAGG';
+JSON_OVERLAPS:                       'JSON_OVERLAPS';
+JSON_PRETTY:                         'JSON_PRETTY';
+JSON_QUOTE:                          'JSON_QUOTE';
 JSON_REMOVE:                         'JSON_REMOVE';
 JSON_REPLACE:                        'JSON_REPLACE';
-JSON_SET:                            'JSON_SET';
-JSON_UNQUOTE:                        'JSON_UNQUOTE';
-JSON_DEPTH:                          'JSON_DEPTH';
-JSON_LENGTH:                         'JSON_LENGTH';
-JSON_TYPE:                           'JSON_TYPE';
-JSON_VALID:                          'JSON_VALID';
-JSON_TABLE:                          'JSON_TABLE';
 JSON_SCHEMA_VALID:                   'JSON_SCHEMA_VALID';
 JSON_SCHEMA_VALIDATION_REPORT:       'JSON_SCHEMA_VALIDATION_REPORT';
-JSON_PRETTY:                         'JSON_PRETTY';
+JSON_SEARCH:                         'JSON_SEARCH';
+JSON_SET:                            'JSON_SET';
 JSON_STORAGE_FREE:                   'JSON_STORAGE_FREE';
 JSON_STORAGE_SIZE:                   'JSON_STORAGE_SIZE';
-JSON_ARRAYAGG:                       'JSON_ARRAYAGG';
-JSON_OBJECTAGG:                      'JSON_OBJECTAGG';
+JSON_TABLE:                          'JSON_TABLE';
+JSON_TYPE:                           'JSON_TYPE';
+JSON_UNQUOTE:                        'JSON_UNQUOTE';
+JSON_VALID:                          'JSON_VALID';
+JSON_VALUE:                          'JSON_VALUE';
+NESTED:                              'NESTED';
```

Comment:
`NESTED` and `ORDINALITY` can be identifiers. Please check other tokens you added and add them to `keywordsCanBeId` rule if necessary.